src/jit/morph.cpp

```diff
@@ -4395,24 +4429,26 @@ void Compiler::fgMorphSystemVStructArgs(GenTreeCall* call, bool hasStructArgumen
                         // Create LCL_FLD for each eightbyte.
                         argListCreated = true;
 
+                        // First field
```

Comment:
OK.

---
In reply to: [79516249](https://github.com/dotnet/coreclr/pull/7252#discussion_r79516249) [](ancestors = 79516249)